django/pyrog/api/serializers/import_export.py

```diff
@@ -153,6 +153,7 @@ class _ResourceSerializer(serializers.ModelSerializer):
     class Meta:
         model = Resource
         fields = [
+            "id",
```

Comment:
be careful: as discussed this `id` field should be ignored in MappingSerializer.create